File: 2116A Gellyfish and Tricolor Pansy.py
t=int(input())
rec=[list(map(int,input().split())) for i in range(t)]
for i in range(t):
    a, b, c, d = rec[i]
    if b <= a and b <= c:
        print("Gellyfish")
    elif a < d:
        print("Flower")
    elif d<=c:
        print("Gellyfish")
    else:
        print("Flower")

File: 2116A Gellyfish and Tricolor Pansy.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() {
    int t;
    cin >> t;
    vector<vector<int>> rec(t, vector<int>(4));
    
    for (int i = 0; i < t; ++i) {
        cin >> rec[i][0] >> rec[i][1] >> rec[i][2] >> rec[i][3];
    }

    for (int i = 0; i < t; ++i) {
        int a = rec[i][0], b = rec[i][1], c = rec[i][2], d = rec[i][3];
        if (b <= a && b <= c) {
            cout << "Gellyfish" << endl;
        }
        else if (a < d) {
            cout << "Flower" << endl;
        }
        else if (d <= c) {
            cout << "Gellyfish" << endl;
        }
        else {
            cout << "Flower" << endl;
        }
    }

    return 0;
}
